packages/serverpod/lib/src/relic/web_server.dart

```diff
@@ -103,7 +103,7 @@ class WebServer {
   }
 
   void _handleRequest(HttpRequest request) async {
-    if (serverpod.runMode == 'production') {
+    if (serverpod.runMode == ServerpodRunMode.production) {
```

Comment:
Comment: Okay to leave this here as that was already present, and this change is an improvement.